src/cr/cube/cube.py

```diff
@@ -653,18 +653,23 @@ def weighted_counts(self):
         _UnweightedCountMeasure object for this cube is returned.
         """
         return (
-            _WeightedCountMeasure(self._cube_dict, self._all_dimensions)
+            _WeightedCountMeasure(
+                self._cube_dict, self._all_dimensions, self._cube_idx_arg
+            )
             if self.is_weighted
-            else _UnweightedCountMeasure(self._cube_dict, self._all_dimensions)
+            else _UnweightedCountMeasure(
+                self._cube_dict, self._all_dimensions, self._cube_idx_arg
```

Comment:
Do we need it in all these instantiations? Or just in `MeanMeasure`? It's gonna be `None` in all of those that we don't need, right?

Reply:
we do need, otherwise we don't swap the summary counts in the exporter:
``` 
E                       5     Dark Knight           73.60           87.67           52.50
E                       6      Fight Club           82.50           93.33           50.00
E                       7  Meets the Par.           23.00            1.00           45.00
E       -               8    Unweighted N              11               7               4
E       +               8    Unweighted N              11               5               4
```